db/upgrade-X.X.X-X.Y.Z.sql

```diff
@@ -11,3 +11,51 @@ CREATE TABLE keyed (
   value LONGBLOB,
   PRIMARY KEY(id)
 ) ENGINE=InnoDB;
+
+--
+-- Table structure for table `iplog_old`
+--
+
+CREATE TABLE iplog_old (
+  mac varchar(255) NOT NULL,
+  ip varchar(255) NOT NULL,
```

Comment:
Same here, we should lower that value to the maximum char length of an ipv4 address